server/views/pages/workAndSkills.njk

```diff
@@ -6,6 +6,9 @@
   }
 ] %}
 {% block body %}
+
+  {% set prisonNamesById = prisonNamesById.value if prisonNamesById.isFulfilled() else {} %}
```

Comment:
Fail gracefully - if the promise is resolved then get the prison ID to name map; else just use an empty object.
The prison name is looked up from this object in `inPrisonCoursesAndQualifications/index.njk` below